ocp_install_env.sh

```diff
@@ -68,15 +68,8 @@ function generate_ocp_install_config() {
 
     outdir="$1"
 
-    deploy_kernel=$(master_node_val 0 "driver_info.deploy_kernel")
-    deploy_ramdisk=$(master_node_val 0 "driver_info.deploy_ramdisk")
-
-    # Always deploy with 0 workers by default.  We do not yet support
-    # automatically deploying workers at install time anyway.  We can scale up
-    # the worker MachineSet after deploying the baremetal-operator
-    #
-    # TODO - Change worker replicas to ${NUM_WORKERS} once the machine-api-operator
-    # deploys the baremetal-operator
+    deploy_kernel=$(node_val 0 "driver_info.deploy_kernel")
+    deploy_ramdisk=$(node_val 0 "driver_info.deploy_ramdisk")
```

Comment:
Is this still used anywhere?  I think we can remove it?

Reply:
https://github.com/openshift-metal3/dev-scripts/pull/903